solutions/07/hypernews.js

```diff
@@ -60,9 +64,9 @@ class Hypernews {
     this.info()
 
     const self = this
-    const view = this.autobase.linearize({
+    this.autobase.start({
       unwrap: true,
-      async apply (batch) {
+      async apply (view, batch) {
         const b = self.bee.batch({ update: false })
```

Comment:
Instead of `self.bee` should be the first argument of the apply

Btw the first arg could be called `bee`, it's often named like that